feat: Add a notebook explaining how to use SSH with Github and naas.ai

Dr0p42 opened the pull request:
The goal of this notebook is to show you how to setup an SSH key with Github and naas.ai.

**What's the benefit?**

You won't have to enter your username/password every time you want to push/pull on a project.

Taking reviews to make sure it's understandable by most peoples 👍 

Here is the notebook: https://github.com/jupyter-naas/awesome-notebooks/blob/naas-github-ssh/Naas/Naas_Configure_Github_with_ssh.ipynb

tdubon commented:
Hi @Dr0p42, I just tried running it and received the following message when executing the second code cell:

zsh:1: no matches found: /home/ftp/.ssh/id_rsa_new*
💡 We need to create an SSH key.
Saving key "/home/ftp/.ssh/id_rsa" failed: No such file or directory

Perhaps it would be helpful to include instruction on how to retrieve the key pair directly. This is the Naas key/token, right?

Dr0p42 commented:
> Hi @Dr0p42, I just tried running it and received the following message when executing the second code cell:
> 
> zsh:1: no matches found: /home/ftp/.ssh/id_rsa_new* 💡 We need to create an SSH key. Saving key "/home/ftp/.ssh/id_rsa" failed: No such file or directory
> 
> Perhaps it would be helpful to include instruction on how to retrieve the key pair directly: https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent

Hello @tdubon, thanks a lot for your review 🙏.

I added a link to the instructions that you referred to :) 

I also removed an `!rm` command that was a left over from my testing and I also added `!mkdir -p /home/ftp/.ssh` to automate the ssh key creation.

Let me know what you think but you should not have the error anymore 🤞
